File-Formats/PDF/Convert-HTML-To-PDF/features.md

```diff
@@ -2187,4 +2187,100 @@ Refer to the following code snippet to set the timeout for HTML to PDF conversio
 
 {% endtabs %}
 
-You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PDF-Examples/tree/master/HTML%20to%20PDF/Blink/Time-out-support-in-HTML-to-PDF/.NET).
\ No newline at end of file
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PDF-Examples/tree/master/HTML%20to%20PDF/Blink/Time-out-support-in-HTML-to-PDF/.NET).
+
+## Accessible PDF
+
+The Blink HTML converter support preserving tags from HTML to PDF using EnableAccessibilityTags property in BlinkConverterSettings class. Refer to the following code snippet.
+
+N> This support fully depends on the Chromium headless browser. Our converter preserves the tags, same as the saved PDF from Chrome.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+	//Initialize HTML to PDF converter.
+	HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
+	//Initialize the BlinkConverterSettings
+	BlinkConverterSettings settings = new BlinkConverterSettings();
+	//Set true to enable the accessibility tags in PDF generation.
+	settings.EnableAccessibilityTags= true;
+	//Assign the BlinkConverterSettings to the ConverterSettings property of HtmlToPdfConverter.
+	htmlConverter.ConverterSettings = settings;
+	//Convert URL to PDF
+	PdfDocument document = htmlConverter.Convert("Input.html");
+	//Save and close the PDF document.
+	FileStream fileStream = new FileStream("HTML-to-PDF.pdf", FileMode.CreateNew, FileAccess.ReadWrite);	
+	document.Save(fileStream);
+	document.Close(true);
+	
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+
+	'Initialize HTML to PDF converter.
+	Dim htmlConverter As New HtmlToPdfConverter()
+	'Initialize the BlinkConverterSettings
+	Dim settings As New BlinkConverterSettings()
+	'Set true to enable the accessibility tags in PDF generation.
+	settings.EnableAccessibilityTags = True
+	'Assign the BlinkConverterSettings to the ConverterSettings property of HtmlToPdfConverter.
+	htmlConverter.ConverterSettings = settings
+	'Convert URL to PDF
+	Dim document As PdfDocument = htmlConverter.Convert("Input.html")
+	'Save and close the PDF document.
+	Dim fileStream As New FileStream("HTML-to-PDF.pdf", FileMode.CreateNew, FileAccess.ReadWrite)	
+	document.Save(fileStream)
+	document.Close(True)
+
+{% endhighlight %}
+
+{% endtabs %}
+
+You can download a complete working sample from [GitHub]().
+
+## Image Background
+
+The Blink HTML converter support adding the image background from HTML to Image using ImageBackgroundColor property in BlinkConverterSettings class. Refer to the following code snippet.
```

Comment:
The Blink HTML converter support adding the image background from HTML to Image using the ImageBackgroundColor property in BlinkConverterSettings class. Refer to the following code snippet.